trades.py

```diff
@@ -53,9 +53,12 @@
 FPMM_CREATOR = "0x89c5cc945dd550bcffb72fe42bff002429f46fec"
 DEFAULT_FROM_DATE = "1970-01-01T00:00:00"
 DEFAULT_TO_DATE = "2038-01-19T03:14:07"
+DEFAULT_FROM_TIMESTAMP = 0
+DEFAULT_TO_TIMESTAMP = 2147483647
```

Comment:
Why? Shouldn't it be "latest"?

Reply:
It's the largest timestamp to call the Omen subgraph. I used the maximum value of unix32 timestamp